documentation/solution-architecture.md

```diff
@@ -0,0 +1,151 @@
+
+The process for issuing proofs as described above is illustrated in verifiable claim sequence diagrams below:
+
+![](images/vcseq1.png)
+
+![](images/vcseq2.png)
+
+
+![](images/vcseq3.png)
+
+![](images/vcseq4.png)
+
+## Application layer
+
+The solution is mainly hosted on a central Discipl ephemeral server application hosted by the issuer to which
+multiple client applications (for clients, issuer and administrator) connect to (through the client side Discipl core api).
+
+The issuer runs a waardepapieren service on this server that sets up the ephemeral server and connects to this to attend to needs expressed by other clients on this platform. The following other client applications connect to this ephemeral service hosted by the issuer:
+
+- clerk-frontend
+- authorisation-service
+- client-frontend
+- adminstrator-frontend
+
+Validation does not require usage of this server; validators run their own embedded local ephemeral service (without client-server communication) just to hold information temporarily and use it for their own validation process executed within a client side validation app. As NLX certificates in the end are to originate from official root certificates (conform EiDAS/EIF), validation for now just involves standard techniques around X.509 certificates and can even be done offline.
+
+Below is a more detailed component view:
+
+![](images/components.png)
+
+## Infrastructural layer
+
+![](images/deployment.png)
+
+The solution is hosted on server(s) in control of the issuer and used by the devices (broad range of desktop computers, mobile phones or tablets) in control of citizens, clerks, validators and administrators. The server at the issuer is accessible on the intranet of the issuer and publicly through an existing web server (in a controlled way through a reverse proxy) that also will serve the client side web application.
+
+To enable easy integration, the waardepapieren service along with a nlx outway are available as docker containers as well as examples of integrations of the clerk and client front end applications. All these can be started using docker compose. The service and front end applications are configurable from a single configuration file. The amount of code to create to integrate the solution in to website / backoffice
+is made as minimal as possible.
+
+## Key decisions
+
+- scope : the waardepapieren service will at least provide in a single product that can be issued.
+- scope : the information being signed is that of the result data of a NLX call given the BSN number as argument though this argument is configurable
```

Comment:
This misses the transformation that we define

Reply:
changed / added mentioning that the information being signed is a configurable selections of fields from the NLX result at two places in the document.